examples/C/src/rosace/fault-tolerance/RosaceWithReExecution.lf

```diff
@@ -0,0 +1,48 @@
+/**
+ * This example runs the Aircraft Simulator with a faulty RosaceController from
+ * "RosaceControllerWithReExecution.lf". Additionally, when it fails, it re-executes from the
+ * checkpoint.
+ *
+ * @author Edward A. Lee
+ * @author David Saussie
+ * @author Claire Pagetti
+ * @author Dongha Kim
+ * @author Hokeun Kim
+ */
+target C {
+  fast: true,
+  build: "../build_run_plot.sh RosaceWithReExecution ../rosace.gnuplot",
+  timeout: 10 min
+}
+
+import Aircraft from "../AircraftSimulator.lf"
+import Command from "../Rosace.lf"
+import FaultTolerantController from "RosaceControllerWithReExecution.lf"
+import PrintToFile from "../../lib/PrintToFile.lf"
+
+preamble {=
+  // Shared constants.
+  // Trimming parameters
+  #define Va_eq (230.0)   // Nominal airspeed?
```

Comment:
What does this comment mean?

Reply:
Hmm. I think it was an old comment from the original Rosace.lf code, but I'll fix it.